### PR TITLE
St 778 migration setup

### PR DESCRIPTION
**Pull request by ericHgorski**

### CODE REVIEW

**Review by mwmerz — APPROVED** [review]

File: src/pages/multisig/utils/placeholder.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 export const SAMPLE_ENCODED_TX =
-  "CpEBCo4BChwvY29zbW9zLmJhbmsudjFiZXRhMS5Nc2dTZW5kEm4KLHRlcnJhMTk1bHFqcXJqZHE3bWZkenl5Zmw2Y3RuOHZ0a3d0NDNndnl1dGxhEix0ZXJyYTE5NWxxanFyamRxN21mZHp5eWZsNmN0bjh2dGt3dDQzZ3Z5dXRsYRoQCgV1bHVuYRIHMTAwMDAwMBIVEhMKDQoFdWx1bmESBDE4NzIQ0IoK"
+  "CpEBCo4BChwvY29zbW9zLmJhbmsudjFiZXRhMS5Nc2dTZW5kEm4KLHRlcnJhMTk1bHFqcx(JqZHE3bWZkenl5Zmw2Y3RuOHZ0a3d0NDNndnl1dGxhEix0ZXJyYTE5NWxxanFyamRxN21mZHp5eWZsNmN0bjh2dGt3dDQzZ3Z5dXRsYRoQCgV1bHVuYRIHMTAwMDAwMBIVEhMKDQoFdWx1bmESBDE4NzIQ0IoK"
```
Comment (terran6) [inline src/pages/multisig/utils/placeholder.ts:2]: Did you mean to add the `(` in the first line here?

CpEBCo4BChwvY29zbW9zLmJhbmsudjFiZXRhMS5Nc2dTZW5kEm4KLHRlcnJhMTk1bHFqcx  (  JqZHE3bWZke

**Review by terran6 — CHANGES_REQUESTED** [review]

**Review by terran6 — CHANGES_REQUESTED** [review]
This branch also results in a dependency issue when executing `npm i`.

**Review by terran6 — CHANGES_REQUESTED** [review]
Still getting dependency issues.

Screenshot 2023-09-05 at 11 51 32 PM

**Comment by ericHgorski** [comment]
> Still getting dependency issues.
> 
> Screenshot 2023-09-05 at 11 51 32 PM

does it work now?

**Comment by terran6** [comment]
> > Still getting dependency issues. The added ts-expect-errors are in componenets that arent's currently used and will continue to be unused in v3
> > Screenshot 2023-09-05 at 11 51 32 PM
> 
> does it work now?

It still doesn't work for me.  It works on your end?

**Comment by ericHgorski** [comment]
> > > Still getting dependency issues. The added ts-expect-errors are in componenets that arent's currently used and will continue to be unused in v3
> > > Screenshot 2023-09-05 at 11 51 32 PM
> > 
> > 
> > does it work now?
> 
> It still doesn't work for me. It works on your end?

`hmm yea, rm -rf node_modules && npm i && npm run build` works fine for me.

**Comment by terran6** [comment]
> rm -rf node_modules && npm i

tried to remove node_modules and install and got the same error:

image

**Comment by ericHgorski** [comment]
> > rm -rf node_modules && npm i
> 
> tried to remove node_modules and install and got the same error:
> 
> image

ok I updated recharts, ok now? Im not sure why we get different deps issues.

**Comment by mwmerz** [comment]
> > rm -rf node_modules && npm i
> 
> tried to remove node_modules and install and got the same error:
> 
> image

What version of node are you using for this project? 

I'm using `16.14.2` for this project. Performing all prescribed steps above builds fine:

1. `rm -rfd node_modules ; rm -rfd *.zip ; rm -rfd build`
2. `npm i`
3. `npm run build`

The normal build pattern, `./build.sh`, which performs `npm ci` to clear modules, works perfectly fine too.

**Comment by terran6** [comment]
> > > rm -rf node_modules && npm i
> > 
> > 
> > tried to remove node_modules and install and got the same error:
> > image
> 
> ok I updated recharts, ok now? Im not sure why we get different deps issues.

After pulling down the recent changes it works fine.

File: src/txs/Tx.tsx [hunk 2]
```diff
@@ -25,23 +25,25 @@ import { isBroadcastingState, latestTxState } from "data/queries/tx"
...
+import { FormError, Select, Input, FormItem } from "components/form"
 import { Modal } from "components/feedback"
 import { Details } from "components/display"
 import { Read } from "components/token"
 import ConnectWallet from "app/sections/ConnectWallet"
 import useToPostMultisigTx from "pages/multisig/utils/useToPostMultisigTx"
 import { isWallet, useAuth } from "auth"
 import { PasswordError } from "auth/scripts/keystore"
-
+import { Submit } from "components/form"
```
Comment (terran6) [inline src/txs/Tx.tsx:37]: Any reason this was taken from the grouped imports from `components/form`?

File: src/txs/Tx.tsx [hunk 3]
```diff
@@ -313,10 +315,10 @@ function Tx<TxValues>(props: Props<TxValues>) {
     return (
       <button
         type="button"
-        className={classNames({ muted: !isMax })}
+        className={cx({ muted: !isMax })}
         onClick={onClick ? () => onClick(max) : () => setIsMax(!isMax)}
       >
-        <Flex gap={4} start>
+        <Flex gap={4}>
```
Comment (terran6) [inline src/txs/Tx.tsx:321]: Something is causing a `Something went wrong` error on the `Send` page.

**Review by terran6 — CHANGES_REQUESTED** [review]

File: src/txs/Tx.tsx (hunk 3 above)
Comment (ericHgorski) [reply src/txs/Tx.tsx:321]: whats the error? not getting this on my end

File: src/txs/Tx.tsx (hunk 2 above)
Comment (ericHgorski) [reply src/txs/Tx.tsx]: Was related to testing. Reverted now.

File: src/txs/Tx.tsx (hunk 3 above)
Comment (terran6) [reply src/txs/Tx.tsx:320]: This issue doesn't seem to do with this branch.  Created a ticket.

**Review by terran6 — APPROVED** [review]

**Comment by ericHgorski** [comment]
should this be PR'd into sprint 25? @mwmerz

**Pull request merged**